examples/rococo/assetHub/foreignAssetTransfers/teleport/assetHubWETHToBridgeHub.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
```

Comment:
Better to remove this example since it will always fail in submission. Bridge hub can't receive any token beyond DOT.

Reply:
In general, system parachains will not be able to receive foreign assets.